src/VirtualClient/VirtualClient.Dependencies/MsMPIInstallation.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace VirtualClient.Dependencies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO.Abstractions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.DependencyInjection;
+    using VirtualClient.Common;
+    using VirtualClient.Common.Extensions;
+    using VirtualClient.Common.Telemetry;
+    using VirtualClient.Contracts;
+
+    /// <summary>
+    /// Installation component for the MS-MPI.
+    /// </summary>
+    public class MSMPIInstallation : VirtualClientComponent
```

Comment:
Name of file and name of class should be identical. Use MsmpiInstallation as the name following standard .NET naming conventions.